Arithmetic/07.GravityCalculatorTests/ProgramTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using GravityCalculator;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GravityCalculator.Tests
+{
+    [TestClass()]
+    public class ProgramTests
+    {
+        [TestMethod()]
+        public void CalcFinalPositionTest_inputParams_outputFallDistance()
```

Comment:
To, ka padod parametru ir skaidrs...bet kas tur specifisks? Derīgi, nederīgi dati?